.travis.yml

```diff
@@ -27,6 +28,10 @@ before_install:
   - gem update --system
   - gem update bundler
 
+before_script:
+  # Patch for Ruby 2.4.0 compatibility.
+  - ./scripts/patch_simplecov_0_12_0
+
```

Comment:
See also https://github.com/lemurheavy/coveralls-ruby/blob/v0.8.19/coveralls-ruby.gemspec#L23
If `coveralls` new version with `simplecov` 0.13 will be released, we do not need this patch for `simplecov`.
It's also good to watch `coveralls` project for that.